examples/spmm/spmm.cc

```diff
@@ -23,129 +24,27 @@ using namespace ttg;
 
 #if defined(BLOCK_SPARSE_GEMM) && defined(BTAS_IS_USABLE)
 using blk_t = btas::Tensor<double>;
+
+// declare btas::Tensor serializable by Boost
+#include "ttg/serialization/backends/boost.h"
+namespace ttg::detail {
+  template <typename Archive>
+  inline static constexpr bool is_boost_serializable_v<Archive, blk_t> = is_boost_archive_v<Archive>;
```

Comment:
Why not simplify this to 

```
inline static constexpr bool is_boost_serializable_v<blk_t> = true;
```

Why is that dependent on the Archive?

Reply:
by definition serializability is Archive-dependent. e.g. user could limit serialization to specific Archive (or class of Archives), could limit it to output archive only, etc.